go.mod

```diff
@@ -33,10 +37,11 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.0
+	github.com/tigera/api v0.0.0-20210803092542-1ba9f4618d05
```

Comment:
Is that right?  I'd expect only projectcalico/api in OSS Felix.

Reply:
Good catch.  Copied the code out of felix-private and forgot that I needed to change that